sdk/fleet/Azure.ResourceManager.ContainerServiceFleet/src/Generated/AutoUpgradeProfileResource.cs

```diff
@@ -349,5 +354,81 @@ public virtual ArmOperation<AutoUpgradeProfileResource> Update(WaitUntil waitUnt
                 throw;
             }
         }
+
+        /// <summary>
+        /// A long-running resource action.
+        /// <list type="bullet">
+        /// <item>
+        /// <term>Request Path</term>
+        /// <description>/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerService/fleets/{fleetName}/autoUpgradeProfiles/{autoUpgradeProfileName}/generateUpdateRun</description>
+        /// </item>
+        /// <item>
+        /// <term>Operation Id</term>
+        /// <description>AutoUpgradeProfileOperations_GenerateUpdateRun</description>
+        /// </item>
+        /// <item>
+        /// <term>Default Api Version</term>
+        /// <description>2025-03-01</description>
+        /// </item>
+        /// </list>
+        /// </summary>
+        /// <param name="waitUntil"> <see cref="WaitUntil.Completed"/> if the method should wait to return until the long-running operation has completed on the service; <see cref="WaitUntil.Started"/> if it should return after starting the operation. For more information on long-running operations, please see <see href="https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/LongRunningOperations.md"> Azure.Core Long-Running Operation samples</see>. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        public virtual async Task<ArmOperation<GenerateResult>> GenerateUpdateRunAutoUpgradeProfileOperationAsync(WaitUntil waitUntil, CancellationToken cancellationToken = default)
```

Comment:
I think `GenerateUpdateRun` could be a better name.
We could do this by adding
```
override-operation-name:
  AutoUpgradeProfileOperations_GenerateUpdateRun: GenerateUpdateRun
```
to your autorest.md file and then regenerate everything to apply it